main.js

```diff
@@ -239,9 +239,12 @@ function createWindow() {
 
     view = new BrowserView({
         webPreferences: {
-            nodeIntegration: true,
-            webviewTag: true,
-            enableRemoteModule: true,
+            nodeIntegration: false,
```

Comment:
When performing the test with these modifications, the buttons injected into the interface no longer appear.

![image](https://user-images.githubusercontent.com/2112638/91463529-e0403f00-e861-11ea-9bac-d4411b8ad633.png)

Reply:
Hm they appeared on my local, let me double check.  We're there any errors in your devtools for the BrowserView?

Reply:
![image](https://user-images.githubusercontent.com/2112638/91471164-0c60bd80-e86c-11ea-82ef-a3250c1356f9.png)